spring-social-zotero/src/main/java/org/springframework/social/zotero/api/ZoteroUpdateItemsStatuses.java

```diff
@@ -0,0 +1,33 @@
+package org.springframework.social.zotero.api;
+
+import java.util.List;
+
+public class ZoteroUpdateItemsStatuses {
+    private List<String> successItems;
+    private List<String> failedItems;
+    private List<String> unchagedItems;
```

Comment:
I would add an error code, so if there is a failure, we know why (e.g. items have changed so the submitted version is older than the server version). Otherwise, it'll be hard to give the user proper feedback on how to resolve the issue.

Reply:
Okay, that makes sense.